validation/src/main/java/io/micronaut/validation/validator/DefaultValidator.java

```diff
@@ -132,7 +132,9 @@ protected DefaultValidator(
         final Collection<BeanProperty<Object, Object>> cascadeProperties =
                 ((BeanIntrospection<Object>) introspection).getIndexedProperties(Valid.class);
 
-        if (CollectionUtils.isNotEmpty(constrainedProperties) || CollectionUtils.isNotEmpty(cascadeProperties)) {
+        final Class<? extends Annotation> pojoConstraint = introspection.getAnnotationTypeByStereotype(Constraint.class).orElse(null);
```

Comment:
This line needs to use `getAnnotationValuesByType` to get all constraints applying to the POJO

Reply:
`introspection.getAnnotationValuesByType(Constraint.class)` returns an empty list. I think I need to use `introspection.getAnnotationTypesByStereotype(Constraint.class)`